pymathics/graph/base.py

```diff
@@ -397,31 +391,24 @@ def is_mixed_graph(self):
     def is_multigraph(self):
         return isinstance(self.G, (nx.MultiDiGraph, nx.MultiGraph))
 
-    def get_sort_key(self, pattern_sort=False) -> tuple:
+    @property
+    def element_order(self) -> tuple:
         """
-        Returns a particular encoded list (which should be a tuple) that is used
-        in ``Sort[]`` comparisons and in the ordering that occurs
-        in an M-Expression which has the ``Orderless`` property.
-
-        See the docstring for element.get_sort_key() for more detail.
+        Return a value which is used in ordering elements
+        of an expression. The tuple is ultimately compared lexicographically.
         """
-
-        if pattern_sort:
-            return super(Graph, self).get_sort_key(True)
-        else:
-            # Return a sort_key tuple.
-            # but with a `2` instead of `1` in the 5th position,
-            # and adding two extra fields: the length in the 5th position,
-            # and a hash in the 6th place.
-            return [
-                1,
-                3,
-                self.class_head_name,
-                tuple(),
-                2,
-                len(self.vertices),
-                hash(self),
-            ]
+        # Return the precedence the expression `Image[]`,
```

Comment:
@mmatera: I adjusted this comment based on https://github.com/Mathics3/mathics-core/blob/master/mathics/builtin/image/base.py#L120-L131, but as I look at this, the comment is a little opaque. 

Why do we want to change "1" to 2"?